api/resources_portal/views/import_materials.py

```diff
@@ -0,0 +1,88 @@
+from django.forms.models import model_to_dict
+from django.http import JsonResponse
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from resources_portal.models import Grant, Material, Organization
+from resources_portal.views.import_sra_utils import ENA_URL_TEMPLATE, gather_all_metadata
+
+
+def import_sra(run_accession, organization, grant, user):
+    """
+    This function returns a Response object containing the json representation of the newly-created material object
+    made using SRA data.
+    """
+    metadata = gather_all_metadata(run_accession)
+
+    # note for the PR: currently, I use "library-construction-protocol" to populate the description. This is a technical description of the steps taken to process the material.
+    # Should I use the study abstract to populate it instead?
```

Comment:
Probably another good question for @cgreene, but I would think the study abstract because these datasets will be at the study/experiment level rather than sample level.

Reply:
I think study/experiment abstract - yea.

Reply:
Changed it to abstract.